modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/CommonAnalysisPlugin.java

```diff
@@ -371,7 +371,7 @@ public TokenStream create(TokenStream tokenStream) {
             if (indexSettings.getIndexVersionCreated().onOrAfter(org.elasticsearch.Version.V_8_0_0)) {
                 throw new IllegalArgumentException(
                     "The [edgeNGram] tokenizer name was deprecated in 7.6. "
-                        + "Please use the tokenizer name to [edge_nGram] for indices created in versions 8 or higher instead."
+                        + "Please use the tokenizer name to [edge_ngram] for indices created in versions 8 or higher instead."
```

Comment:
While we're at it, can you also change use with "change" here? 
```suggestion
                        + "Please change the tokenizer name to [edge_ngram] for indices created in versions 8 or higher instead."
```